staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -219,6 +219,12 @@ type UpdatedObjectInfo interface {
 	// UpdatedObject returns the updated object, given a context and old object.
 	// The only time an empty oldObj should be passed in is if a "create on update" is occurring (there is no oldObj).
 	UpdatedObject(ctx genericapirequest.Context, oldObj runtime.Object) (newObj runtime.Object, err error)
+
+	// ValidateCreate applies validating admission to a newly created object.
+	ValidateCreate(obj runtime.Object) error
```

Comment:
@liggitt you're the original author.  Yay or nay?  I don't feel strongly one way or the other.

Reply:
The name `ObjectInfo` does not match anymore with this change. It's more an `UpdateInfo` now.